product_docs/docs/pem/9/pem_rel_notes/900_rel_notes.mdx

```diff
@@ -6,7 +6,7 @@ New features, enhancements, bug fixes, and other changes in PEM 9.0.0 include:
 
 | Type        | Description                                                                                                                                                                                                             | ID       |
 | ----------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------   | -------- |
-| Enhancement | Refreshed look and feel, adding space to the layout of tables and other screens and introducing updated EDB branding and accent colors.                                                                                 | PEM-4472 |
+| Enhancement | Added space to the layout of tables and other screens to have new look and feel. Also updated EDB branding and accent colors.                                                                                 | PEM-4472 |
```

Comment:
Improved layout of the user interface and updated with new branding.

Reply:
done